templates/download/cloud/build-openstack.html

```diff
@@ -0,0 +1,226 @@
+{% extends "download/_base_download.html" %}
```

Comment:
Another page being created here - is this the same thing? Should this remain deleted 'cos it's not in the new IA?